ZelBack/config/default.js

```diff
@@ -1,6 +1,6 @@
-const userconfig = require('../../config/userconfig');
+let userconfig = require('../../config/userconfig'); // eslint-disable-line
 
-const isDevelopment = userconfig.initial.development || false;
+const isDevelopment = userconfig.initial.development || false; // eslint-disable-line
 
```

Comment:
what is the  // eslint-disable-line for? if not needed please remove it.

Reply:
without it you won't compile home ui and it will throw an error